### DISCUSSION TITLE
v.3.0.5

### DISCUSSION THREAD

**Discussion by assafelovic** [assafelovic/gpt-researcher#769] [Announcements]
This week includes a critical update to all GPTR users - we've focused on optimizations and happy to update that GPTR pip package is now **87% lighter** (!!). This is big news to all Docker users and has been a long time request. In addition, we've added support for human in the loop to the multi agent solution. This itself marks a new milestone toward better human-agent experience throughout the research processes. Thank you again to the #1 AI community!

## What's Changed
* Stabilize master by @ElishaKay in https://github.com/assafelovic/gpt-researcher/pull/755
* Optimized project dependencies by @assafelovic in https://github.com/assafelovic/gpt-researcher/pull/756
* minimalistic-deps by @ElishaKay in https://github.com/assafelovic/gpt-researcher/pull/757
* made use of azure_openai consistent (and not just in the base model, … by @danieldekay in https://github.com/assafelovic/gpt-researcher/pull/759
* WIP: Human in the loop - frontend support by @ElishaKay in https://github.com/assafelovic/gpt-researcher/pull/762

## New Contributors
* @danieldekay made their first contribution in https://github.com/assafelovic/gpt-researcher/pull/759

**Full Changelog**: https://github.com/assafelovic/gpt-researcher/compare/v3.0.4...v.3.0.5

<hr /><em>This discussion was created from the release <a href='https://github.com/assafelovic/gpt-researcher/releases/tag/v.3.0.5'>v.3.0.5</a>.</em>

**Comment by danieldekay** [comment]
:) Love it. My very very first ever pull request to an open source project. Happy to help!
